src/app.rs

```diff
@@ -25,7 +25,17 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Include hidden directories and files in the search results (default: \
                          hidden files and directories are skipped). Files and directories are \
-                         considered to be hidden if their name starts with a `.` sign (dot).",
+                         considered to be hidden if their name starts with a `.` sign (dot). \
+                         Flag can be overridden with --no-hidden.",
+                ),
+        )
+        .arg(
+            Arg::with_name("no-hidden")
+                .long("no-hidden")
+                .overrides_with("no-hidden")
```

Comment:
Shouldn't it also override `--hidden`? We don't want `--hidden` and `--no-hidden` to be present at the same time, right? This way, you might even be able to simplify the logic in `main.rs` (not completely sure).

Reply:
I thought we did want to support both of them being present; at least that's the use case mentioned in the original issue [here](https://github.com/sharkdp/fd/issues/595#issue-619793513) with aliasing.

Reply:
Ah, yes. Sorry for that. My comment was confusing. What I meant was: I think we can let `clap` help us here. If we set `no-hidden` "`overrides_with`" `hidden`, I think that `clap` internally either has "hidden" or "no-hidden" present, never both.

We definitely want the user to be able to pass both of them.

Reply:
Ah, I see what you mean now. Yep, I think that'll clean up `main.rs` a bit.